### PR TITLE
Create data loader for exported POPROX data

### PR DESCRIPTION
**Pull request by karlhigley**

### CODE REVIEW

**Comment by mdekstrand** [comment]
Over in #127, to reduce overhead in measurement, I added a method `user_has_history(self, user: UUID) -> bool` that just checks if a user has history; the measurement code was constructing entire requests just to see if the user had history (and therefore had personalized recommendations).

However, whether the recommendations are personalized seems like something we should actually be recording at recommendation time, so I'm not super happy with that solution.

**Comment by mdekstrand** [comment]
Actually, never mind that — I can do a cleaner fix in #127.

**Review by mdekstrand — APPROVED** [review]
LGTM

**Review by sophiasun0515 — APPROVED** [review]
looks good!

File: src/poprox_recommender/evaluation/evaluate.py [hunk 1]
```diff
@@ -25,75 +29,94 @@
 from docopt import docopt
 from progress_api import make_progress
 
+# from poprox_recommender.data.mind import MindData
```
Comment (karlhigley) [inline src/poprox_recommender/evaluation/evaluate.py:32]:
```suggestion
```

**Pull request merged**